dev/phpunit/unit/Ampersand/PatchHelper/Patchfile/SanitiserTest.php

```diff
@@ -111,4 +111,59 @@ public function testStripCommentsFromPhp()
             file_get_contents(BASE_DIR . '/dev/phpunit/unit/resources/patchfile/sanitiser/php/FooBarBazExpected.php')
         );
     }
+
+    /**
+     * @dataProvider sanitisePhtmlCasesDataProvider
+     */
+    public function testSanitisePhtmlCases($input, $expected)
+    {
+        $contents = Sanitiser::sanitisePhtml($input);
+        $this->assertEquals($expected, $contents);
+    }
+
+    public function sanitisePhtmlCasesDataProvider()
+    {
+        return [
+            [
+                "<?php       echo      123      ;  ;  ; \n\n\n ;   ; \n  ;  ; ?>",
```

Comment:
@hostep Wonder what you think of these cases, based on your comment on #13 

---
Some patterns which I think can be ignored:
- whitespace-only changes (can probably be fixed with adding the `-w` flag to the diff command?)
- copyright header changes
- everything which is in comments and has changed (not sure about php docblocks?)
- `<?php echo ...` vs `<?= ...` in phtml files
- extra `;` at the end of a single php line in phtml files
- ...
----

The php copyright header and comments changes are stripped as part of the tokenizer. When its detected as some inline html/js we pass back to the comment stripper functions for those ones.

Then theres a few extra changes to handle `;` and short echos and multiple concurrent spaces 

This is no my prettiest code, but just wondering what you think of these phtml cases covered?

Reply:
Looks good to me!